data_analysis/plots.py

```diff
@@ -0,0 +1,171 @@
+from pathlib import Path
+from typing import Union
+
+import matplotlib
+matplotlib.use('QtAgg')
+
+import logging
+import folium
+import geopandas as gpd
+import pandas as pd
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import compare_scheduled_and_rt
+import static_gtfs_analysis
+
+CHICAGO_COORDINATES = (41.85, -87.68)
+PLOTS_PATH = Path(__file__).parent.parent / 'plots' / 'scratch'
```

Comment:
I wonder if we should make the git repo root path an environment variable and then use that to construct these paths, just to make it more generalizable?

Reply:
That would probably be a better bet. I'll update this.